src/lib/viewers/office/OfficeLoader.js

```diff
@@ -29,14 +30,15 @@ class OfficeLoader extends AssetLoader {
     /**
      * @inheritdoc
      */
-    determineViewer(file, disabledViewers = []) {
+    determineViewer(file, disabledViewers = [], viewerOptions = {}) {
         // The Office viewer is disabled when this is a password protected shared link
         const isDisabledDueToPasswordProtectedSharedLink = file.shared_link && file.shared_link.is_password_enabled;
-        // If the user does not have permission to download the file, the file is larger than 5MB, or isDisabledDueToSharedLink is true,
-        // then disable the Office viewer
+        // If the user does not have permission to download the file, the file is larger than Appconf file size or 5MB,
+        // or isDisabledDueToSharedLink is true, then disable the Office viewer
+        const maxFileSize = get(viewerOptions, 'office.excelOnlineAllowedMaxFileSize') || FIVE_MB;
```

Comment:
Partners team submitted a PR using the name `excelOnlineAllowedMaxFileSize`, so I just used it.

Reply:
I can discuss with them to use a better name.

Reply:
The person opened that PR is on vacation. Should we directly use a reasonable name and tell her to change it in her PR, since her PR has not been merged yet?

Reply:
I think if I remember correct, that PR is still open, so we should do what's right for Preview SDK and reflect that in the usage PR